gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -15,7 +15,7 @@ def self.postgres_dir
     end
 
     def self.postgresql_template
-      RAILS_ROOT.join("../system/TEMPLATE").join(postgres_dir)
+      PostgresAdmin.template_directory.join(postgres_dir)
```

Comment:
It's pretty complicated but the TEMPLATE directory has stuff laid out as they get applied to /, like a chroot.  So, we need to the root of the TEMPLATE directory and the normal PG_DATA directory (postgres_dir) in this case.